install_multiple.yml

```diff
@@ -0,0 +1,4 @@
+import: install.yml
+
```

Comment:
Maybe we should call this file `install_plugin`, since this will be used specifically to test plugins.